src/System.Private.CoreLib/shared/System/Environment.cs

```diff
@@ -18,6 +19,16 @@ public static partial class Environment
         /// </summary>
         internal static bool IsSingleProcessor => ProcessorCount == 1;
 
+        internal static int ProcessorIdModCount
```

Comment:
This is only used in one place. It does not seem to be worth putting it into a separate method.

Reply:
Thinking its just what GetCurrentProcessorId should return?

Reply:
Moved `GetCurrentProcessorId` to shared